test/unit/helpers/contacts/RecipientsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace SendGrid;
+
+class RecipientsTest_Recipient extends \PHPUnit_Framework_TestCase
+{
+    public function testRecipientsForm()
+    {
+        $form = new RecipientForm('http://www.examle.com/recipientFormSubmit');
```

Comment:
Maybe it would make much more sense if we could cast this object to string to have a representation of `RecipientForm` in string, e.g:

(string) new RecipientForm('http://www.examle.com/recipientFormSubmit');

Reply:
Makes sense- will do.